src/SME.SGP.Aplicacao/CasosDeUso/EncaminhamentoAee/ObterEncaminhamentoPorIdUseCase.cs

```diff
@@ -42,7 +42,17 @@ public async Task<EncaminhamentoAEERespostaDto> Executar(long id)
                 Situacao = encaminhamentoAee.Situacao,
                 PodeEditar = podeEditar,
                 MotivoEncerramento = encaminhamentoAee.MotivoEncerramento,
-                Auditoria = (AuditoriaDto)encaminhamentoAee
+                Auditoria = (AuditoriaDto)encaminhamentoAee,
+                responsavelEncaminhamentoAEE = new ResponsavelEncaminhamentoAEEDto()
+                {
+                    Id = encaminhamentoAee.ResponsavelId == null ? 0 : encaminhamentoAee.ResponsavelId,
+                    Nome = encaminhamentoAee.ResponsavelNome == null ? "" 
+                    :
+                    string.IsNullOrEmpty(encaminhamentoAee.ResponsavelNome) ? "" : encaminhamentoAee.ResponsavelNome,
+                    Rf = encaminhamentoAee.ResponsavelRf == null ? ""
+                    :
+                    string.IsNullOrEmpty(encaminhamentoAee.ResponsavelRf) ? "" : encaminhamentoAee.ResponsavelRf,
+                }
```

Comment:
Carregar do atribuito Responsavel
```suggestion
                Auditoria = (AuditoriaDto)encaminhamentoAee,
                responsavelEncaminhamentoAEE = encaminhamentoAee.Responsavel == null ? null :
                new ResponsavelEncaminhamentoAEEDto()
                {
                    Id = encaminhamentoAee.Responsavel.Id,
                    Nome = encaminhamentoAee.Responsavel.Nome,
                    Rf = encaminhamentoAee.Responsavel.CodigoRf
                }
```